From d7fc8b8073629063d37569ae9f3c4de09856ed13 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 1 Nov 2012 16:27:24 +0800
Subject: [PATCH 091/310] ENGR00232023 sabresd: set the default lcd density to
 160dpi

As the ipu fb driver has the ability to export
the screen panel width/height size to userspace.
The LVDS we used is actually 128dpi, calculated
by the gralloc framebuffer settings. But
this density breaks the display quality on
our LVDS. So keep the 160dpi still makes sense.

Change-Id: I75da8c54506ff0760a99354c2cee0cde90c1dea1
Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/sabresd/init.rc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/imx6/sabresd/init.rc b/imx6/sabresd/init.rc
index d686c73..1d4dcbc 100755
--- a/imx6/sabresd/init.rc
+++ b/imx6/sabresd/init.rc
@@ -45,6 +45,9 @@ on boot
     # Set rotation to 270 to cofigure as portrait mode
     setprop ro.sf.hwrotation 0
 
+    # Set the density to 160dpi, default 128dpi is not good
+    setprop ro.sf.lcd_density 160
+
 # change for FSL specific service
     chown system system /sys/devices/system/cpu/cpu0
     chown system system /sys/devices/system/cpu/cpu1
-- 
1.8.0

